### DISCUSSION TITLE
How to install on ubuntu 22.04 ?

### DISCUSSION THREAD

**Discussion by fhazal** [qaisjp/go-discord-irc#145] [Q&A] [answered]
can anyone help me with tutorial how to install go-discord-irc in ubuntu 22.04 server, i can't find anything on how to install in readme. Im new with install this kind of stuff, but know the basic stuff about ssh command, can anyone help me ? 

and how do i configure this ? i know how to add discord channel id, but i don't know how to get irc channel id 

```
# Updating this will automatically add or remove puppets from channels
channel_mappings:
  "#bottest chanKey": 316038111811600387
  "#bottest2": 318327329044561920

```
i can make the bot at discord online but not seeing any discord user in my irc server.

**Comment by qaisjp** [comment, marked as answer]
there are no "channel Ids" on irc. the left side is the name of the irc channel. the chanKey is the password for the channel, but not every irc channel has a password, so the chanKey is optional.

the right side is the discord channel ID

**Reply by qaisjp** [reply]
please let me know if this answers your question

**Reply by fhazal** [reply]
can u make a step how to install for a newbie like me ? i really like the apps but not really sure how to install in ubuntu 22.04? 

how to configure webirc? do i need to have kiwiirc setup on my server? or just need to create a webirc block in unrealircd? in discord irc config just need to put passwd? no other setting?

can i use this with matterbridge?? i got matterbridge running on my server too, but matterbridge dont bring discord user into irc, what is the implication and how to optimize the setup with matterbridge ??
